rosdep/python.yaml

```diff
@@ -5601,6 +5601,10 @@ python3-backoff-pip:
   ubuntu:
     pip:
       packages: [backoff]
+python3-bidict:
+  debian: [python3-bidict]
```

Comment:
Since this package is only available since bullseye, but we still support buster, we need to do something slightly more complicated here.  In particular:

```
debian:
  '*': [python3-bidict]
  buster: null
```